core/contracts/financial_templates/implementation/PricelessPositionManager.sol

```diff
@@ -319,17 +319,18 @@ contract PricelessPositionManager is FeePayer {
             return totalPaid;
         }
 
-        // TODO(#873): add divCeil and mulCeil to make sure that all rounding favors the contract rather than the user.
         // Adjust internal variables below.
         // Compute fee percentage that was paid by the entire contract (fees / pfc).
-        FixedPoint.Unsigned memory feePercentage = totalPaid.div(initialPfc);
+        // TODO(#873): do we want to add add divCeil and mulCeil here to make sure that all rounding favors the contract rather than the user?
+        // Similar issue to _payOracleFees
+        FixedPoint.Unsigned memory feePercentage = totalPaid.divCeil(initialPfc);
 
         // Compute adjustment to be applied to the position collateral (1 - feePercentage).
         FixedPoint.Unsigned memory adjustment = FixedPoint.fromUnscaledUint(1).sub(feePercentage);
 
         // Apply fee percentage to adjust totalPositionCollateral and positionFeeAdjustment.
-        totalPositionCollateral = totalPositionCollateral.mul(adjustment);
-        positionFeeAdjustment = positionFeeAdjustment.mul(adjustment);
+        totalPositionCollateral = totalPositionCollateral.mulCeil(adjustment);
+        positionFeeAdjustment = positionFeeAdjustment.mulCeil(adjustment);
```

Comment:
Yeah good point, the main thing we want to `ceil` is the `div` above so that we ultimately get a lower `adjustment` value, rendering collateral to be lower than expected due to rounding errors